### PR TITLE
Fix #17533: Missing audio when specifying --rct2-data-path

### LINKED ISSUE #17533: Missing audio when specifying --rct2-data-path
### Operating System

Windows 10 Home, 64 bit, 21H1

### OpenRCT2 build

OpenRCT2 0.4.1 Windows Portable x64

### Describe the issue

When I boot up the game it is completely quiet, no menu music. Then when I load a park, the only sounds are the ride music. There are no sounds of guests, rides, anything else.

When I use the stable releases of 0.3.5, 0.3.5.1, and 0.4.0, it all works fine. But as soon as I use the 0.4.1 release, the sound issue comes back. So I don't believe that it is my data files, otherwise the previous versions would be affected too, and they aren't, it's just the latest stable release.

Note that I keep my game data files in separate folders, so that it is a tidier setup and so that when a new version of OpenRCT2 comes out, all I have to do is replace the OpenRCT2 files. I keep four main folders next to each other as follows:

OpenRCT2  -  this contains the OpenRCT2 files
RCT1Data  -  this holds the data files from RCT1
RCT2Data  -  this holds the data files from RCT2
UserData  -  this holds the user data files, like settings, saved games, scenarios, etc.

So to run OpenRCT2, I use a batch file that calls the executable and points it at the various data folders.

START OpenRCT2\openrct2.exe --user-data-path=UserData --openrct2-data-path=OpenRCT2\data --rct1-data-path=RCT1Data --rct2-data-path=RCT2Data
EXIT

I've seen some other people post about sounds issues, but they all seem to be from a few years back and all are resolved.

### Area(s) with issue?

_No response_

### Steps to reproduce

1. Extract the files from the latest release (0.4.1).
2. Provide the RCT1 and RCT2 data files in separate folders.
3. Call the OpenRCT2 executable with a batch script pointing to the various data folders.
4. No error message is received, but the only sounds that play are the ride music. No menu music, no guest sounds, no ride sounds, etc.
5. Restore a previous version of OpenRCT2 (e.g. 0.3.5, 0.3.5.1, 0.4.0) and load it up as above and it works fine again.

### Attachments

_No response_

### PR DESCRIPTION
**Pull request by IntelOrca**

### CODE REVIEW

**Comment by janisozaur** [comment]
What does 'auto' mean here?

File: distribution/changelog.txt [hunk 1]
```diff
@@ -10,6 +10,7 @@
 - Fix: [#17312] (Flying) Inline Twist appearing under the surface when placed on ground level.
 - Fix: [#17444] “Manta Ray” boats slowed down too much in “Ayers Rock” scenario (original bug).
 - Fix: [#17503] Parks with staff with an ID of 0 have all staff windows focus on that staff
+- Fix: [#17533] Missing auto when specifying --rct2-data-path.
```
Comment (IntelOrca) [inline distribution/changelog.txt:13]:
```suggestion
- Fix: [#17533] Missing audio when specifying --rct2-data-path.
```

**Comment by IntelOrca** [comment]
Squash merge

File: distribution/changelog.txt [hunk 2]
```diff
@@ -10,6 +10,7 @@
 - Fix: [#17312] (Flying) Inline Twist appearing under the surface when placed on ground level.
 - Fix: [#17444] “Manta Ray” boats slowed down too much in “Ayers Rock” scenario (original bug).
 - Fix: [#17503] Parks with staff with an ID of 0 have all staff windows focus on that staff
+- Fix: [#17533] Missing audio when specifying --rct2-data-path.
```
Comment (Broxzier) [inline distribution/changelog.txt:13]:
```suggestion
- Fix: [#17533] Missing audio when specifying ‘--rct2-data-path’.
```

**Review by Broxzier — COMMENTED** [review]
Code LGTM, also tested to verify this bug is fixed. Left one small note for a more consistent changelog.

**Review by janisozaur — APPROVED** [review]

**Pull request merged**